ecs-cli/modules/config/readwriter_v1.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package config
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/Sirupsen/logrus"
+)
+
+const (
+	yamlConfigFileName = "config.yml"
+	configFileMode     = os.FileMode(0600)
```

Comment:
nit: it'd be good to document what permissions this translates to as you might have to extend CLI to work on non Linux platform in the future.